types/openui5/README.md

```diff
@@ -42,10 +42,34 @@ The reasoning behind this is that with semantic versioning, the API will remain
 The only exception when we might still create new patch releases at DefinitelyTyped is when there are significant improvements or fixes in the documentation or in the definition generator. 
 
 Long story short:
-* For ease of consumption and most probably matching definitions, use the latest major/minor-matching version of `@types/openui5`.
+* For ease of consumption and most probably matching definitions, use the latest major/minor-matching version of `@types/openui5`. Also see the section below about the referenced jQuery and QUnit types. 
 * For perfectly matching definitions, use the version of [`@openui5/ts-types-esm`](https://www.npmjs.com/package/@openui5/ts-types-esm) which is identical to the used OpenUI5 version.
 
 
+## jQuery and QUnit References and their Versions
+
+OpenUI5 includes copies of jQuery and QUnit and occasionally exposes types from them in its API.
+
+In the DefinitelyTyped infrastructure, we can only specify *that* the OpenUI5 types/APIs depend on the types of jQuery and QUnit, but we cannot specify the respective *version*. So the `package.json` file generated by DefinitelyTyped for the npm release references the `"*"` version (any/latest version), not the version of the jQuery/QUnit copy which is actually included with the OpenUI5 framework.
+
+Hence, as of writing (OpenUI5 version 1.100, March 2022), the versions of the types being installed together with `@types/openui5` are:
```

Comment:
today is the last chance to submit this wording...

Reply:
Tomorrow still works, isn't it?

Reply:
Fixed.